### DISCUSSION TITLE
Why use gloo controller as ELASTIC horovod controller?

### DISCUSSION THREAD

**Discussion by ezioliao** [horovod/horovod#2981] [Q&A] [answered]
I have always wonder why **ELASTIC** horovod use gloo as coordinating controller instead of mpi.  For better performance or other reason? 

Can anyone help me solve my doubts?

**Comment by maxhgerlach** [comment, marked as answer]
You cannot shutdown and re-initialize Horovod running on MPI without starting new processes via mpirun. That's an intrinsic limitations of many (or all?) implementations of MPI.

**Reply by ezioliao** [reply]
Got it! Thanks.
